.env.example

```
@@ -74,7 +74,18 @@ CELERY_ENABLE_UTC=true
+FLOWER_URL_PREFIX=
+FLOWER_MAX_TASKS=10000
 
 # Callback Timeouts (in seconds)
 CALLBACK_CONNECT_TIMEOUT = 3
 CALLBACK_READ_TIMEOUT = 10
+
+
+# EC2 Deployment Configuration for Celery Workers
+EC2_HOST=production-celery-host.example.com
+EC2_USER=ubuntu
+EC2_KEY=production-ec2-key
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The EC2_KEY key should go before the EC2_USER key

(UnorderedKey)